src/test/kotlin/com/ampnet/walletservice/controller/WithdrawControllerTest.kt

```diff
@@ -217,15 +218,69 @@ class WithdrawControllerTest : ControllerTestBase() {
         }
     }
 
+    @Test
+    @WithMockCrowdfoundUser
+    fun mustBeAbleToGetWithdrawForTxHash() {
+        suppose("User has approved withdraw") {
+            testContext.withdraw = createApprovedWithdraw(userUuid, withFile = true)
+        }
+
+        verify("User can get his pending withdraw") {
+            val result = mockMvc.perform(get(withdrawPath))
+                .andExpect(status().isOk)
+                .andReturn()
```

Comment:
Is query param missing here?

Reply:
Yep, forgot to add it, the test passes because it falls back to option of returning all user withdraws.